src/logging/logging.module.ts

```diff
@@ -0,0 +1,48 @@
+import { Global, Module } from '@nestjs/common';
+import { LoggingService } from './logging.interface';
+import { RequestScopedLoggingService } from './logging.service';
+import * as winston from 'winston';
+import * as Transport from 'winston-transport';
+
+/**
+ * Provides a new instance of a Winston logger using the provided {@link transports}
+ *
+ * @param transports - the logger transports to be used in the winston instance
+ */
+function winstonFactory(transports: Transport[] | Transport): winston.Logger {
+  return winston.createLogger({ transports: transports });
+}
+
+const LoggerTransports = Symbol('LoggerTransports');
+
+/**
+ * Factory which provides a collection of transports to be used by the
+ * logger instance
+ */
+function winstonTransportsFactory(): Transport[] | Transport {
+  return new winston.transports.Console({
+    level: 'debug',
```

Comment:
I took this from the Gelato Relay Service, but I think this is a clear candidate for externalization. We could find it useful to have different logging levels between environments, so I think this could be tackled in a separate issue/PR if you are OK with it.

Reply:
Yes that's a good idea. We can set it in the application configuration (we can keep the default as `debug` if none is set).

Reply:
Perfect, I created an issue for this: https://github.com/safe-global/safe-client-gateway-nest/issues/313